common/dbconnector.cpp

```diff
@@ -232,7 +232,12 @@ SonicDBInfo& SonicDBConfig::getDbInfo(const std::string &dbName, const std::stri
         SWSS_LOG_ERROR("%s", msg.c_str());
         throw out_of_range(msg);
     }
-    auto& infos = foundNetns->second;
+    return foundNetns->second;
+}
+
+SonicDBInfo& SonicDBConfig::getDbInfo(const std::string &dbName, const std::string &netns)
+{
```

Comment:
```suggestion
{
    std::lock_guard<std::recursive_mutex> guard(m_db_info_mutex);
    
    SWSS_LOG_ENTER();
    
```

Make sure to hold the lock while we are accessing the `m_db_info` structure. Also, the convention is to log entry into the method.